pkgs/os-specific/linux/joycond-cemuhook/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, python3Packages, fetchFromGitHub}:
+
+python3Packages.buildPythonPackage {
+  pname = "joycond-cemuhook";
+  pyproject = true;
+  version = "unstable-2023-08-09";
+
+  src = fetchFromGitHub {
+    owner = "joaorb64";
+    repo = "joycond-cemuhook";
+    rev = "3c0e07374ff431a0f8ae70dbb0b5a62fb3de06ee";
+    hash = "sha256-K24CEmYWhgkvVX4geg2bylH8TSvHIpsWjsPwY5BpquI=";
+  };
+
+  nativeBuildInputs = with python3Packages; [
+    dbus-python
+    evdev
+    pyudev
+    termcolor
+    setuptools
+    setuptools-git-versioning
+    setuptools-git
+    wheel
+  ];
```

```suggestion
  nativeBuildInputs = with python3Packages; [
    setuptools
    setuptools-git-versioning
    setuptools-git
  ];

  propagatedBuildInputs = with python3Packages; [
    dbus-python
    evdev
    pyudev
    termcolor
  ];
```
See the [docs](https://ryantm.github.io/nixpkgs/languages-frameworks/python/#buildpythonpackage-parameters). In particular, the nativeBuildInputs are used when building the package and the propagatedBuildInputs are the runtime dependencies of the package.

---

Oh huh I didn't know there were docs for this! Thank you! I didn't know how many of the dependencies were runtime or build-time, since I didn't write the original application. I'll fix this.

---

For some Python projects, the dependencies are listed in pyproject.toml: https://github.com/joaorb64/joycond-cemuhook/blob/master/pyproject.toml